crew

```diff
@@ -406,7 +406,16 @@ end
 
 def build_and_preconfigure (target_dir)
   Dir.chdir target_dir do
-    puts "Building from source, this may take a while..."
+    puts "Building from source, this may take a while..."    
+    
+    # Remove /usr/local/lib64/*.la to resolve any "can not find /usr/local/*.la" compile errors when building the packages (only needed for x86_64).
+    # https://gnunet.org/faq-la-files
+    # https://stackoverflow.com/questions/42963653/libquadmath-la-is-not-a-valid-libtool-archive-when-configuring-openmpi-with-g
+    if @opt_verbose then
```

Comment:
What happened to the `if ARCH == 'x86_64'` conditional?

Reply:
Only required fo x86_64

Reply:
Right but now it deletes `*.la` files for all architectures.

Reply:
there are not any errors like "*.la" not a valid libtool archiv